### PR TITLE
Clarify gen-driftignore output destination example

### PR DESCRIPTION
**Pull request by craigfurman**
Can I also change the 0.17 docs and republish them, or do we have to cut new releases to publish new docs?

### CODE REVIEW

**Comment by sjourdan** [comment]
I definitely used to edit the current docs in place (example: https://github.com/snyk/driftctl-docs/pull/64/files)

**Comment by craigfurman** [comment]
Nice, I've edited this to update the 0.17 docs. Will the pipeline automatically update the live site (https://docs.driftctl.com/0.17.0/usage/cmd/gen-driftignore-usage/)?

**Comment by sundowndev** [comment]
> Nice, I've edited this to update the 0.17 docs. Will the pipeline automatically update the live site (https://docs.driftctl.com/0.17.0/usage/cmd/gen-driftignore-usage/)?

Yes, the entire site is built from scratch at each deployment.

**Comment by craigfurman** [comment]
Forgot to mention - this PR is from a @benlaplanche suggestion

File: docs/usage/filtering/driftignore.mdx [hunk 1]
```diff
@@ -44,4 +44,4 @@ The above mechanism to ignore resources can be used in combination with filter r
 
 ## Automatically generate a driftignore file
 
-See [driftignore generator command](../cmd/gen-driftignore-usage).
\ No newline at end of file
+See [driftignore generator command](../cmd/gen-driftignore-usage).
```
Comment (wbeuil) [inline docs/usage/filtering/driftignore.mdx:47]: I don't understand this diff. What's the difference?

Comment (sundowndev) [reply docs/usage/filtering/driftignore.mdx:47]: It took time for me as well, there's a space at the end of the initial line

Comment (craigfurman) [reply docs/usage/filtering/driftignore.mdx:47]: must be my editor auto-trimming whitespace

**Review by wbeuil — APPROVED** [review]
LGTM

**Pull request merged**